examples/fields/SmartDateField.README.md

```diff
@@ -0,0 +1,59 @@
+# SmartDateField
+
+## Why
+
+The `SmartDateField` lets you worry about operations on `Date`s as objects, without worrying about parsing dates, or formatting dates.  Currently the `SmartDateField` can parse 9 date types without any format strings.  This lets you concentrate on business logic, without worrying about incoming date formats.  This let's you confidently perform arithmetic and comparisons in `compute`, `recordCompute`, `batchRecordsCompute`, and `validate` on `Date`s.  `SmartDateField` has an `fString` argument that controls how the date is serialized after `validate` is finished.  Flatfile users no longer have to convert in and out of dates to maintain formatting.
```

Comment:
What are the 9 date types that can be parsed without any format strings?

Reply:
They are visible in tests... I'll add a link